<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjMergeDlg</class>
 <widget class="QDialog" name="ProjMergeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>95</y>
     <width>398</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add Project</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="removeButton">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QListWidget" name="projsListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>46</y>
     <width>323</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Project Folder:</string>
      </property>
      <property name="buddy">
       <cstring>outfolderEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="outfolderEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Project Name:</string>
      </property>
      <property name="buddy">
       <cstring>projnameEdit</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="projnameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>242</y>
     <width>213</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okayButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_2</zorder>
  <zorder>projnameEdit</zorder>
  <zorder>okayButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder></zorder>
 </widget>
 <tabstops>
  <tabstop>projnameEdit</tabstop>
  <tabstop>outfolderEdit</tabstop>
  <tabstop>selectButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>okayButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>projsListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjMergeDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
